### DISCUSSION TITLE
Precompiled versions

### DISCUSSION THREAD

**Discussion by SikVac** [mriscoc/Ender3V2S1#6] [General]
Are you considering releasing precompiled versions ?

**Comment by mriscoc** [comment]
Yes, but only a standard version, no the Anniversary Update until I can fix some licensing and libraries issue.

**Reply by pauluchin** [reply]
Hola... consulta de que me perdi y que diferencias hay entre la versión aniversario y esta ultima ... llegue tarde apara ocupar la anterior :) saludos desde un país vecino... gracias por tu tiempo invertido en este pedazo de firmware...

**Reply by mriscoc** [reply]
Hola. La versión que a la fecha está publicada aquí tiene menos características que la edición de aniversario.

**Comment by Okywan** [comment]
Hola,

Alguna opción para conseguir versiones precompiladas? Aunque sea la estandar. No consigo compilar con VSCode y Auto Build Marlin...

Gracias.

**Reply by mriscoc** [reply]
Puede descargar ahora las versiones pre compiladas de la página de [releases](https://github.com/mriscoc/Ender3V2S1/releases/latest)

**Comment by jeroenvermeulen** [comment]
Build instructions: #12
